website/docs/getting-started/cli.md

```diff
@@ -16,6 +16,61 @@ This can be done by running the following command:
 xattr -d com.apple.quarantine /path/to/osc
 ```
 
+## Usage in GitHub Actions
+
+When using `osc` in a GitHub action, the [`setup-osc`](https://github.com/eclipse-apoapsis/setup-osc) action can be used to install `osc` and authenticate with an ORT Server instance.
+
+The following example demonstrates how to integrate `osc` in a GitHub Actions workflow to start an ORT run, retrieve the created reports, and store them as workflow artifacts:
+
+```yaml
+jobs:
+  run-osc:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Download OSC and authenticate
+      uses: eclipse-apoapsis/setup-osc@main
+      with:
+        osc-version: 0.1.0-RC16
+        base-url: https://ort-server.example.com
+        token-url: https://auth.example.com/realms/prod/protocol/openid-connect/token
+        client-id: ort-server-prod
+        username: user
+        password: ${{ secrets.ORT_SERVER_PASSWORD }}
+
+    - name: Start ORT run
+      run: |
+        osc runs start --parameters '{
+          "revision": "${{ github.head_ref || github.ref_name }}",
+          "jobConfigs": {
+            "analyzer": {},
+            "advisor": {},
+            "evaluator": {},
+            "scanner": {},
+            "reporter": {},
+            "notifier": {}
+          }
+        }'
+      env:
+        OSC_REPOSITORY_ID: 42
+        OSC_RUNS_START_WAIT: true
```

Comment:
I think this variable is not explained anywhere else on the page, maybe add a short comment explaining what it does?